hypothesis-python/tests/conjecture/common.py

```diff
@@ -201,7 +202,9 @@ def draw_float_kwargs(
         min_value = draw(st.floats(max_value=pivot, allow_nan=False))
 
     if use_max_value:
-        min_val = min_value if not pivot is not None else max(min_value, pivot)
+        min_val = (
+            min_value if not pivot is not None else sign_aware_max(min_value, pivot)
```

Comment:
Maybe drop the double negation - `if pivot is None`?

The code around float handling is so subtle, I had to really think to be confident that there were no hidden pitfalls :-)

Otherwise, :100:

Reply:
no idea how I ended up writing that 😆